simplesat/dependency_solver.py

```diff
@@ -16,7 +16,7 @@
 
 class DependencySolver(object):
     def __init__(self, pool, remote_repositories, installed_repository,
-                 policy=None, use_pruning=True):
+                 policy=None, use_pruning=True, prefer_installed=True):
```

Comment:
I would prefer putting that choice in the policy class, to avoid polluting the signature of `DependencySolver`.